hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocationServiceSupport.java

```diff
@@ -169,7 +169,9 @@ public boolean isShutdown() {
     public void shutdown() {
         isShutdown = true;
         responseThread.interrupt();
-        callIdMap.clear();
+        CleanResourcesTask cleanResourcesTask = new CleanResourcesTask();
+        cleanResourcesTask.run();
```

Comment:
In this call, there seems to be unnecessary wait logic: waitForPacketsProcessed

Can we just call ClientInvocationServiceSupport::cleanResources directly?